jdaviz/core/launcher.py

```diff
@@ -82,7 +86,12 @@ def _launch_config_with_data(config, data=None, show=True, **kwargs):
 
     # Load data
     if data not in (None, ''):
-        viz_helper.load_data(data, **kwargs)
+        try:
+            viz_helper.load_data(data, **kwargs)
+        except IORegistryError:
+            if filepath is None:
+                raise
+            viz_helper.load_data(filepath, **kwargs)
```

Comment:
Not the prettiest thing I've ever seen but I don't hate it. And it works, which is the important thing!

Reply:
agreed... hopefully we can revert this down the road, but that may require some significant refactoring [:cat:](https://jira.stsci.edu/browse/JDAT-3755)